src/CRUD/CRUD.go

```diff
@@ -0,0 +1,145 @@
+package main
+
+import (
+	"bytes"
+	"flag"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"github.com/Comcast/webpa-common/wrp"
+)
+
+const (
+	AuthHeaderName				= "Authorization"
+	TypeHeaderName              = "X-Xmidt-Message-Type"
+	ContentHeaderName           = "X-Xmidt-Content-Type"
+	TransIdHeaderName           = "X-Xmidt-Transaction-Uuid"
+	SourceHeaderName            = "X-Xmidt-Source"
+	DestHeaderName 				= "X-Webpa-Device-Name"
+	PathURL 					= "https://api.xmidt.comcast.net/api/v2/device"
+)
+
+
+var auth string
+
+func makeRequest(requestType, messageType, source, transId, dest, contentType, payload string, client *http.Client) {
+
+	
+	headers := make(map[string]string)
+	
+	headers[AuthHeaderName] = auth
+	var req *http.Request
+	var err error
+
+	if "POST" == requestType {
+		headers[TypeHeaderName]= messageType
+		headers[ContentHeaderName]= contentType
+		headers[TransIdHeaderName]= transId
+		headers[SourceHeaderName]= source
+		headers[DestHeaderName]= dest
+		
+		req, err = http.NewRequest(requestType, PathURL, bytes.NewBufferString(payload))
+	}
+	
+
+	if err != nil {
+		fmt.Printf("Request %s failed: %s, %v\n", requestType, PathURL, err)
+	}
+
+	for k, v := range headers {
+		req.Header.Set(k, v)
+	}
+	
+	resp, err := client.Do(req)
+	
+	if err != nil {
+		fmt.Printf("Failed to obtain request: %v\n", err)
+	} else {
+
+		if 200 != resp.StatusCode {
+			fmt.Printf("Failed. resp.StatusCode %v \n", resp.StatusCode)
+		} else if 200 == resp.StatusCode {
+			/*Decode Msgpack response to JSON format*/
+			target := wrp.AllFormats()
+			decoder := wrp.NewDecoder(resp.Body, target[0])
+			var buffer bytes.Buffer
+			encoder := wrp.NewEncoder(&buffer, target[1])
+
+			if message, err := wrp.TranscodeMessage(encoder, decoder); err != nil {
+				fmt.Println("Error while converting:", target[0], "to", target[1], err)
+			} else {
+				//Headers
+				fmt.Println("Content-Type :", resp.Header.Get("Content-Type"))
+				fmt.Println("Status :", resp.Header.Get("X-Xmidt-Status"))
+				fmt.Println("Build :", resp.Header.Get("X-Scytale-Build"))
+				fmt.Println("Flavor :", resp.Header.Get("X-Scytale-Flavor"))
```

Comment:
In print use same name as header X-Scytale-Flavor
Add constants for header names X-Scytale-Flavor